README.md

```diff
@@ -36,63 +42,106 @@ These microservices are written in [Python 3](https://www.python.org/) and shoul
 
 How to install and run each microservice for local development.
 
+Each microservice has slightly different dependencies and thus slightly different installation procedures. Generally, installation for each microservice is as follows, but each subfolder's README.md has an "Installing" section with more accurate instructions.
+
 Make sure your working directory is the microservice's subfolder. For example, for the "users" microservice:
 
-```
+```sh
 cd users
 ```
 
 Set up and activate a virtual environment.
 
-```
+```sh
 python3 -m venv venv && . venv/bin/activate
 ```
 
 Install the required python modules.
 
-```
+```sh
 pip3 install -r requirements.txt
 ```
 
-Provision a MongoDB instance (e.g. via [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)), then provide the app with its endpoint via an environmental variable.
+Provision a MongoDB instance (e.g. via [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)), then provide the app with its endpoint via an environmental variable (this step is not needed for the pageserve service).
```

Comment:
Same here - I think it's better to use 'environment' that 'environmental'.

Reply:
Done https://github.com/knative-portability/large-events/pull/39/commits/0027abd27173b1107cad33ce2aebba06ae606027